ts/components/settings/section/CategoryConversations.tsx

```diff
@@ -81,13 +80,48 @@ const AudioMessageAutoPlaySetting = () => {
   );
 };
 
+const EnterKeyFunctionSetting = () => {
+  const forceUpdate = useUpdate();
+
+  const initialSetting = window.getSettingValue(SettingsKey.settingsEnterKeyFunction) || true;
+
+  const items = [
+    {
+      label: window.i18n('enterSendNewMessageDescription'),
+      value: 'enterSend',
+    },
+    {
+      label: window.i18n('enterNewLineDescription'),
+      value: 'enterNewLine',
+    },
+  ];
```

Comment:
I have a few comments to do, but first I'd like to know why not just use a boolean for that setting?
It would be easier to track (as it would be tracked in redux) and easier to default to another value.
What other option can we have for that setting except the "enter sends" or the "enter makes new line"?